tjpcov/covariance_fourier_ssc.py

```diff
@@ -78,13 +78,16 @@ def get_covariance_block(
         tr[3], tr[4] = tracer_comb2
 
         cosmo = self.get_cosmology()
-        mass_def = ccl.halos.MassDef200m()
-        hmf = ccl.halos.MassFuncTinker08(cosmo, mass_def=mass_def)
-        hbf = ccl.halos.HaloBiasTinker10(cosmo, mass_def=mass_def)
+        mass_def = ccl.halos.MassDef200m
+        hmf = ccl.halos.MassFuncTinker08(mass_def=mass_def)
```

Comment:
Not sure, why we use Tinker08 as default instead of Tinker10. I am using Tinker10 for ng.  If there is no obvious reason to use Tinker08, we can keep this (so we don't break the current benchmarks) and I open an issue for keeping track of this

Reply:
No idea. I would leave it like this for this PR to avoid mixing changes